public/static/gameScene.js

```diff
@@ -60,13 +66,31 @@ class GameScene extends Phaser.Scene {
             }),
         });
 
+        this.anims.create({
+            key: "laserFlux",
+            frameRate: 8,
+            repeat: -1,
+            frames: this.anims.generateFrameNames("laser", {
+                start: 0,
+                end: 2,
+            }),
+        });
+
+
+
+        //Load socket
+        this.socket = io(ENDPOINT, { query: "purpose=game" });
```

Comment:
Creating a socket in gameScene.js no longer applies as of the synchronize PR, since the socket is created in game.js and is passed along all scenes. Removing this line will probably resolve the error.

Reply:
yeah removing that line is fine, thanks